WalletWasabi.Fluent/Views/Wallets/Send/PrivacyWarningsView.axaml

```diff
@@ -112,7 +112,8 @@
           <Border Classes="warning">
             <DockPanel>
               <PathIcon />
-              <TextBlock Text="Transaction interlinks labels:" Margin="0 0 5 0" />
+              <TextBlock Text="Transaction interlinks labels:" Margin="0 0 5 0"
+                         ToolTip.Tip="Multiple entities can see this transaction. You should coinjoin first or carefully select who should know about this transaction with the label management tool (if available)." />
```

Comment:
I wouldn't say "can see" because anyone can see this transaction on the blockchain.
The following is shorter and more precise:

```suggestion
                         ToolTip.Tip="Those entities know this is your transaction. Coinjoin more or use the label management tool to carefully select who would know that this transaction is yours." />
```